app/ee/omnichannel/containers/OmnichannelStatus.tsx

```diff
@@ -20,16 +20,19 @@ interface IOmnichannelStatus {
 }
 
 const OmnichannelStatus = memo(({ searching, goQueue, queueSize, inquiryEnabled, user }: IOmnichannelStatus) => {
-	if (searching || !(RocketChat.isOmnichannelModuleAvailable() && user?.roles?.includes('livechat-agent'))) {
-		return null;
-	}
 	const { theme } = useTheme();
-	const [status, setStatus] = useState(isOmnichannelStatusAvailable(user));
+	const [status, setStatus] = useState<boolean>(false);
 
 	useEffect(() => {
-		setStatus(isOmnichannelStatusAvailable(user));
+		if (RocketChat.isOmnichannelModuleAvailable() && user?.roles?.includes('livechat-agent')) {
```

Comment:
This new code is repeating `RocketChat.isOmnichannelModuleAvailable() && user?.roles?.includes('livechat-agent')`.
We should refactor to reuse it.


This could be enough:
```tsx
const canUseOmnichannel = RocketChat.isOmnichannelModuleAvailable() && user?.roles?.includes('livechat-agent')
```